### PR TITLE
Create newFile

### PR DESCRIPTION
**Pull request by mileseverest**

### CODE REVIEW

File: newFile [hunk 1]
```diff
@@ -0,0 +1 @@
+new
```
Comment (mileseverest) [inline newFile]: review

Comment (mileseverest) [inline newFile]: review

Comment (mileseverest) [reply newFile]: reply

Comment (milese123) [inline newFile]: comment1

Comment (milese123) [inline newFile]: test

Comment (milese123) [inline newFile]: test

**Comment by milese123** [comment]
issue

**Comment by milese123** [comment]
comment

**Comment by milese123** [comment]
commentlkjadf

**Comment by mileseverest** [comment]
hi

**Pull request merged**